rotkehlchen/tests/unit/decoders/test_hop.py

```diff
@@ -101,20 +175,519 @@ def test_hop_optimism_eth_receive_no_event(database, optimism_inquirer, optimism
+def test_hop_usdc_bridge_l2_to_l1_gnosis(database, gnosis_inquirer: 'GnosisInquirer', gnosis_accounts):  # noqa: E501
+    tx_hash = deserialize_evm_tx_hash('0x0c6084399c873b06407f073acec89ffd9693ac0c7df4befd9e45e4178b5ae869')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=gnosis_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp, bridge_amount, gas_fees = TimestampMS(1715204760000), '60786.713495', '0.0001299435'
+    user_address = gnosis_accounts[0]
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.GNOSIS,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_XDAI,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label=user_address,
+            notes=f'Burned {gas_fees} XDAI for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=462,
+            timestamp=timestamp,
+            location=Location.GNOSIS,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=Asset(A_HUSDC_GNOSIS),
+            balance=Balance(amount=FVal(bridge_amount)),
+            location_label=user_address,
+            notes=f'Burn {bridge_amount} of Hop hUSDC',
+            counterparty=CPT_HOP,
+            address=ZERO_ADDRESS,
+        ),
+    ]
+    assert events == expected_events
+
+
+# @pytest.mark.vcr(ilter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x0e414c1c4780df6c09c2f1070990768D44B70b1D', '0xA63734db2c674122EEd383aea7698C68aAbf749e']])  # noqa: E501
+def test_hop_eth_bridge_arbitrum_custom_recipient(
+        database, arbitrum_one_inquirer: 'ArbitrumOneInquirer', arbitrum_one_accounts,
+):
+    tx_hash = deserialize_evm_tx_hash('0x88b6fce15afacab9fa77caaf8d42d30d7a517f9f5ae6b7d6c37795309ac90383')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=arbitrum_one_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp, gas, bridge_amount = TimestampMS(1715946201000), '0.00000266018', '0.000880551717232904'  # noqa: E501
+    hop_fee = '0.000118862618614123'
+    user_address = arbitrum_one_accounts[0]
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas)),
+            location_label=user_address,
+            notes=f'Burned {gas} ETH for gas',
+            counterparty=CPT_GAS,
+        ),
+        EvmEvent(
```

Comment:
again please don't do this. Keep together here and in the entire file

```suggestion
        ), EvmEvent(
```